themes/custom/govcms8_uikit_starter/styleguide/item-ui-kit.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html class="js" lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>GovCMS 8 UI-Kit Starter Style Guide</title>
+
+  <meta name="description" content="">
+  <meta name="generator" content="kss-node">
+  <meta name="viewport" content="width=device-width">
+
+  <link rel="stylesheet" href="kss-assets/kss.css">
+  <link rel="stylesheet" href="kss-assets/govcms8_kss.css">
+  <link rel="stylesheet" href="../css/style.css">
+
+</head>
+<body id="kss-node" class="kss-fullscreen-mode">
+
+<div class="kss-sidebar au-body au-body--dark">
+  <header class="kss-header">
+    <h1 class="kss-doc-title">GovCMS 8 UI-Kit Starter Style Guide</h1>
+  </header>
+  <nav class="kss-nav">
+    <ul class="kss-nav__menu">
```

Comment:
This menu needs a link to take you back to the main site.

Is it ok to use "../../../../../../"?